### PR TITLE
Fix Code Block Formatting

### LINKED ISSUE #5442: Fix code formatting.
The README.md file of [docs/code-samples/eventing/helloworld/helloworld-go/README.md](https://github.com/knative/docs/blob/main/code-samples/eventing/helloworld/helloworld-go/README.md)
## Expected Behavior
The code Blocks should have the highlighted formatting. Also should not show ```go  
knDocBefore

## Actual Behavior
Should show the highlighted formatting. and only the corresponding code.

## Steps to Reproduce the Problem

1. Have to move the code block. Indentation problem

### PR DESCRIPTION
**Pull request by debasishbsws**
Fixes #5442

## Proposed Changes 

- move the indentation for proper code highlighting.

Now it is looking fine

knDocAfter

### CODE REVIEW

**Review by psschwei — COMMENTED** [review]
Thanks for the PR!
/lgtm
/approve

**Comment by psschwei** [comment]
/hold
there's some trailing whitespace that needs to be fixed

**Comment by debasishbsws** [comment]
Fixed white spaces

File: code-samples/eventing/helloworld/helloworld-go/README.md [hunk 1]
```diff
@@ -360,85 +362,87 @@ Play around with the CloudEvent attributes in the curl command and the trigger s
...
-        name: event-display
-        namespace: knative-samples
-      spec:
-        replicas: 1
-        selector:
-          matchLabels: &labels
+1.  Using the following example, create a YAML file for a Pod that receives any
+    CloudEvent and logs the event to its output:
+
+        ```yaml
```
Comment (psschwei) [inline code-samples/eventing/helloworld/helloworld-go/README.md:368]: the indent on this one is incorrect: https://github.com/knative/docs/blob/9ea2b2fc6488979243217cafcda96ec1c875bc75/code-samples/eventing/helloworld/helloworld-go/README.md#verify-reply-from-helloworld-go-app

File: code-samples/eventing/helloworld/helloworld-go/README.md [hunk 2]
```diff
@@ -360,85 +362,87 @@ Play around with the CloudEvent attributes in the curl command and the trigger s
...
-             source: knative/eventing/samples/hello-world
-         subscriber:
-           ref:
-             apiVersion: v1
-             kind: Service
-             name: event-display
-       ```
-
-1. Apply the YAML file by running the command:
+1.  Using the following example, create a YAML file for a trigger to deliver the event
```
Comment (psschwei) [inline code-samples/eventing/helloworld/helloworld-go/README.md:413]: this one too

Comment (debasishbsws) [reply code-samples/eventing/helloworld/helloworld-go/README.md:413]: fixd

**Review by psschwei — COMMENTED** [review]
Thanks!

/lgtm
/hold cancel

**Pull request merged**